set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -414,16 +415,16 @@ public UserSet fetchItems(UserSet userSet, String sort, String sortOrder, int pa
 	}
 
 	String apiKey = getConfiguration().getSearchApiKey();
-	String url = buildSearchApiUrl(userSet, apiKey, sort, sortOrder, pageNr, pageSize);
-//    	uri = userSet.getIsDefinedBy() + additionalParameters;
-
-	SearchApiResponse apiResult;
+	String url = buildSearchApiPostUrl(userSet, apiKey);
+	SearchApiRequest searchApiRequest = buildSearchApiPostBody(userSet, pageSize);
```

Comment:
the request must not ignore pageNr, sort and sortOrder parameters

Reply:
done.